data_handler/migrations/versions/efd02f93572b_health_ratio_model_adjusted.py

```diff
@@ -18,7 +18,7 @@
 
 # revision identifiers, used by Alembic.
 revision: str = "efd02f93572b"
-down_revision: Union[str, None] = None
+down_revision: str = "509baf9251f2"
```

Comment:
Did you change it or it was changed automatically during rebasing/merging branches?

Reply:
Yes I did it, because there was a problem with migrations (I couldn't run it) and this solution worked.